src/pages/profile/Profile.tsx

```diff
@@ -0,0 +1,56 @@
+import { FC, useEffect, useState } from "react";
+import profilepic from "../../assets/profile.jpg";
+import { Button } from "@/common/button";
+import { Error } from "@/common/error";
+import { useParams } from "react-router-dom";
+
+export const Profile : FC = ()=>{
```

Comment:
<Profile> will contain more components than you think
split it to `features/user` (we will have avatar/follow user card/ etc)

head up to [preprod](https://github.com/blogify-app/blogify-web/blob/preprod) to see how it was made for auth